src/main/java/org/gridsuite/explore/server/services/ContingencyListService.java

```diff
@@ -49,15 +52,16 @@ public void setActionsServerBaseUri(String actionsServerBaseUri) {
         this.actionsServerBaseUri = actionsServerBaseUri;
     }
 
-    public Mono<Void> deleteContingencyList(UUID id) {
+    public Mono<Void> delete(UUID id, String userId) {
         String path = UriComponentsBuilder.fromPath(DELIMITER + ACTIONS_API_VERSION + "/contingency-lists/{id}")
             .buildAndExpand(id)
             .toUriString();
 
         return webClient.delete()
             .uri(actionsServerBaseUri + path)
+            .header(HEADER_USER_ID, userId)
```

Comment:
userId isn't used in actions-server on this endpoint. 
You added it cause you think it gonna be used later ?

Reply:
to have the same interface everywhere, and yes possibly use it (trace)

Reply:
Ok